
Showing logs for build [32mmeta-seldonio-seldon-core-pr-14-k924d-1[0m stage [32mmeta-pipeline[0m and container [32mstep-credential-initializer-s5mk8[0m
{"level":"warn","ts":1582192717.7086737,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1582192717.7096775,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-14-k924d-1[0m stage [32mmeta-pipeline[0m and container [32mstep-working-dir-initializer-qqwg8[0m
{"level":"warn","ts":1582192718.6687667,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1582192718.6713786,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-14-k924d-1[0m stage [32mmeta-pipeline[0m and container [32mstep-place-tools[0m

Showing logs for build [32mmeta-seldonio-seldon-core-pr-14-k924d-1[0m stage [32mmeta-pipeline[0m and container [32mstep-git-source-meta-seldonio-seldon-core-pr-14-wtq2k[0m
{"level":"warn","ts":1582192802.7631788,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1582192808.2911184,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ c09c05533f1b07f34766fb315b58f19703a9e72e in path /workspace/source"}

Showing logs for build [32mmeta-seldonio-seldon-core-pr-14-k924d-1[0m stage [32mmeta-pipeline[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:c09c05533f1b07f34766fb315b58f19703a9e72e,1466:4f26373fb427e4628966cdd5a888871e661bf992
DEBUG: ran git fetch --unshallow origin 4f26373fb427e4628966cdd5a888871e661bf992: c09c05533f1b07f34766fb315b58f19703a9e72e: in 
DEBUG: ran git checkout master in 
DEBUG: ran git reset --hard c09c05533f1b07f34766fb315b58f19703a9e72e in 
DEBUG: ran clean --force -d . in 
error: error during merge: merging 4f26373fb427e4628966cdd5a888871e661bf992 into master: git output: Auto-merging helm-charts/seldon-core-analytics/Chart.yaml
CONFLICT (content): Merge conflict in helm-charts/seldon-core-analytics/Chart.yaml
Automatic merge failed; fix conflicts and then commit the result.: failed to run 'git merge 4f26373fb427e4628966cdd5a888871e661bf992' command in directory '', output: 'Auto-merging helm-charts/seldon-core-analytics/Chart.yaml
CONFLICT (content): Merge conflict in helm-charts/seldon-core-analytics/Chart.yaml
Automatic merge failed; fix conflicts and then commit the result.'
[31m
Pipeline failed on stage 'meta-pipeline' : container 'step-git-merge'. The execution of the pipeline has stopped.[0m
